frontend/src/metabase/core/hooks/use-form-error-message/use-form-error-message.ts

```diff
@@ -1,11 +1,11 @@
-import { useLayoutEffect, useState } from "react";
+import { useContext, useLayoutEffect, useState } from "react";
 import { useFormikContext } from "formik";
 import { t } from "ttag";
-import useFormState from "metabase/core/hooks/use-form-state";
+import FormContext from "metabase/core/context/FormContext";
 
 const useFormErrorMessage = (): string | undefined => {
   const { values, errors } = useFormikContext();
-  const { status, message } = useFormState();
+  const { status, message } = useContext(FormContext);
```

Comment:
Should we maybe add another hook like `useForm` that'd basically `return useContext(FormContext);`? So there's no need to carry over the context here and there all the time

Reply:
Fixed - added a hook called `useFormContext`. Decided to use the same naming convention the formik does, i.e. `useFormikContext`